landing-style.css

```
@@ -0,0 +1,171 @@
+@import url(./color.css);
+
+/* landing page css */
+
+.header {
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 1.25rem 12.5rem;
+  background: var(--linkbrary-bg);
```

```suggestion
  background-color: var(--linkbrary-bg);
```
저는 backgroung-color를 줄때는 background보다는 더 명확한 요소로 주는 편이에요. 왜냐하면 background는 color말고도 다른 요소가 있어서 해당 프로퍼티를 읽을떄는 이 요소엔 왜 color만 있지라는 의문이 생기더라구요,